bandersnatch/multiexp.go

```diff
@@ -448,37 +478,48 @@ func (p *PointProj) msmC4(points []PointAffine, scalars []fr.Element, splitFirst
 	// critical for performance
 
 	// each go routine sends its result in chChunks[i] channel
-	var chChunks [nbChunks]chan PointProj
-	for i := 0; i < len(chChunks); i++ {
-		chChunks[i] = make(chan PointProj, 1)
-	}
-
-	processChunk := func(j int, points []PointAffine, scalars []fr.Element, chChunk chan PointProj) {
+	//var chChunks [nbChunks]chan PointProj
```

Comment:
unused code chunk here

Reply:
I left these comments here since this was still a draft PR to get a temperature check of the proposed changes. 
I'll clean this for sure :+1: